profile/README.md

```diff
@@ -0,0 +1,17 @@
+# freeCodeCamp
+
+<img src="https://cdn.freecodecamp.org/platform/universal/fcc_meta_1920X1080-indigo.png" align="center" alt="freeCodeCamp's logo" />
+
+[freeCodeCamp.org](https://www.freecodecamp.org) is a friendly community where you can learn to code for free. It is run by a [donor-supported 501(c)(3) nonprofit](https://www.freecodecamp.org/donate) to help millions of busy adults transition into tech.
+
+Our full-stack web development and machine learning curriculum is completely free and self-paced. We have thousands of interactive coding challenges to help you expand your skills.
+
+You can [read more about freeCodeCamp](https://freecodecamp.org/news/about).
+
+## Our Curriculum
+
+The codebase for our primary `/learn` platform can be found in the [freeCodeCamp repository](https://github.com/freeCodeCamp/freeCodeCamp), which includes our curriculum lessons, web client, and API server.
+
+## Contributing
+
+If you are interested in contributing to freeCodeCamp, we recommend starting with our [contributing documentation](https://contribute.freecodecamp.org).

```

Comment:
I think we should remove these. Instead, let's use this feature to have a call for sponsors. I would use the copy and the list of projects from out GitHub sponsors page

Reply:
If you go that route this is a cool tool to generate a SVG that we can reference in the content: https://github.com/antfu/sponsorkit

Reply:
I went ahead and added the sponsors copy.